hw/hw2-baseball alternate method table 3.sql

```diff
@@ -0,0 +1,35 @@
+-- Alternate method for third table in BDA_696 HW2
+-- WORKS FOR MYSQL ONLY, WILL NOT RUN IN MARIADB
+-- Significantly faster than self-join method
+
+-- BEFORE RUNNING, disable full group by in mySQL:
+-- SET GLOBAL sql_mode=(SELECT REPLACE(@@sql_mode,'ONLY_FULL_GROUP_BY',''));
+
+USE baseball;
+
+DROP TABLE IF EXISTS rolling_avg_day_faster
```

Comment:
need a semicolon after this statement, but otherwise it looks good.

Reply:
Good catch! Thanks!